code-of-conduct.md

```diff
@@ -0,0 +1,73 @@
+Amaç
+-------
+
+Ruby Türkiye Topluluğunun (rubytr olarak da bilinir) birincil amacı, mümkün olan en çeşitli ve farklı geçmişlere sahip en çok sayıda katılımcıyı kapsamaktır. Bu nedenle, cinsiyet, cinsel yönelim, engellilik hali, etnik köken, sosyoekonomik statü ve dini ne olursa olsun herkes için samimi, güvenli ve hoş bir ortam sağlamayı taahhüt ediyoruz.
+
+Bu Davranış Kuralları, topluluğumuza katılan herkes için beklentilerimizin yanı sıra kabul edilemez davranışların sonuçlarını özetlemektedir.
+
+
+Açık [Kaynak/Kültür/Teknoloji] Vatandaşlığı
+--------------------------------------
+
+Bu Davranış Kurallarının amacı, katılımcıları eylemlerimiz ve onların toplumumuz üzerindeki etkileri arasındaki ilişkileri tanımaya ve güçlendirmeye teşvik ederek açık [kaynak/kültür/teknoloji] vatandaşlığını arttırmaktır.
+
+Topluluklar, içinde bulundukları toplumları yansıtırlar ve toplumda var olan birçok eşitsizlik ve güç ihlallerine karşı koymak için olumlu eylemler şarttır.
+
+Topluluğumuzda sıcak, samimi ve tüm katılımcıları tam anlamıyla katkıda bulunmaya teşvik etmek için ekstra çaba harcayan birini görürseniz, bilmek isteriz.
```

Comment:
bu cumleyi anlamadim.

Reply:
`If you see someone who is making an extra effort to ensure our community is welcoming, friendly, and encourages all participants to contribute to the fullest extent, we want to know.` su cumlenin cevirisi aslinda ama simdi bakinca ben de anlam veremedim. Silebiliriz bence, ne dersin?

Reply:
Kaldıralım bence de

Reply:
Anlam kargaşası yaşatmamak ve daha sade ve anlaşılır olması için cümleleleri azaltabildiğimiz kadar azaltmalıyız.